.github/workflows/build-main.yml

```diff
@@ -6,7 +6,11 @@ on:
 
 jobs:
   build-main:
-    name: Build and push a main snapshot image
+    name: Build and push main snapshot images
+    strategy: 
+      matrix:
+        architecture: [amd64,ppc64le,s390x]
```

Comment:
ubi8 image is built also for arm64(aarch64)
```
[root@localhost ~]# skopeo inspect --raw docker://registry.access.redhat.com/ubi8-micro:8.5 | jq .manifests[].platform
{
  "architecture": "amd64",
  "os": "linux"
}
{
  "architecture": "arm64",
  "os": "linux"
}
{
  "architecture": "ppc64le",
  "os": "linux"
}
{
  "architecture": "s390x",
  "os": "linux"
}
```
and the same for ubi9 beta
```
[root@localhost ~]# skopeo inspect --raw docker://registry.access.redhat.com/ubi9-beta/ubi-micro:9.0.0beta-455 | jq .manifests[].platform
{
  "architecture": "amd64",
  "os": "linux"
}
{
  "architecture": "arm64",
  "os": "linux"
}
{
  "architecture": "ppc64le",
  "os": "linux"
}
{
  "architecture": "s390x",
  "os": "linux"
}
```

Reply:
Thanks. will update that.

So we also need other os supported e.g. darwin (mac) or windows?

@tonytcampbell FYI

Reply:
Thank you for changes. 

I hope it works as expected cause sometime there are different names for architectures in different context
* amd64 vs x86_64
* arm64 vs aarch64

I do not feel suitable for review cause I am not sure which semantic should be used in this context.
I would be guessing I let other review the code.

But lgtm from my POV

Reply:
Thanks @lslebodn